.travis.yml

```diff
@@ -6,7 +6,7 @@ language: node_js
 matrix:
   include:
     - os: linux
-      node_js: 8
+      node_js: 8.9.3
```

Comment:
Is this what electron uses  in 2.0.8?

Reply:
Hard to tell, their version numbers in files are all over the place but that is the one they used at the start of 2.0.x

Reply:
there is a table now https://github.com/electron/node#branching-strategy